### PR TITLE
Finished Google Hangouts feature

### PR DESCRIPTION
**Pull request by adnanhemani**
This is not production-ready code at this time. Needs refactoring.

### CODE REVIEW

**Comment by an-ju** [comment]
The UI looks good to me.

Is the large hangout logo a placeholder for something else?

**Comment by an-ju** [comment]
screen shot 2018-08-01 at 4 22 54 pm

Uploaded a screenshot for reference.

File: app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -85,6 +120,39 @@ def select
     end
   end
 
+  def getHangoutsLink
```
Comment (an-ju) [inline app/controllers/dashboard_controller.rb]: Style: could you please change this to `get_hangouts_link` to stay consistent with the convention?  You may also need to update some other places.

File: app/controllers/dashboard_controller.rb [hunk 2]
```diff
@@ -85,6 +120,39 @@ def select
...
+        })
+
+        $service.request_options.retries = 5
+        result = $service.insert_event($calendar_id, event, conference_data_version: 1)
+
+        proj.event_id = result.id
+        proj.save!
+      }
+    end
+    redirect_to "/"
```
Comment (an-ju) [inline app/controllers/dashboard_controller.rb]: `redirect_to root_path` might be better.

File: app/controllers/dashboard_controller.rb [hunk 3]
```diff
@@ -85,6 +120,39 @@ def select
     end
   end
 
+  def getHangoutsLink
+    query = Project.where(pivotal_id: params[:project_id].to_s)
+    if (!query.any?) 
+      proj = Project.new
```
Comment (an-ju) [inline app/controllers/dashboard_controller.rb]: This seems like a class method of `Project`.

File: app/models/project.rb [hunk 4]
```diff
@@ -1,2 +1,7 @@
-class Project
-end
\ No newline at end of file
+class Project < ActiveRecord::Base
+    def self.getHangoutJoinIcon(project)
```
Comment (an-ju) [inline app/models/project.rb]: `get_hanghout_join_icon`

File: spec/models/project_spec.rb [hunk 5]
```diff
@@ -1,4 +1,5 @@
-require 'spec_helper'
+require 'rails_helper'
 
-describe Project, type: :model do
-end
\ No newline at end of file
+RSpec.describe Project, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (an-ju) [inline spec/models/project_spec.rb]: Please add some tests when the implementation is done.

**Review by an-ju — COMMENTED** [review]
There're some style/convention suggestions.  I generally follow the suggestions given by my RubyMine IDE, and I think it makes maintenance easier and the codebase cleaner.

**Comment by adnanhemani** [comment]
So the logo is actually a link. I do need to get it to be smaller though

File: app/controllers/dashboard_controller.rb (hunk 2 above)
Comment (adnanhemani) [reply app/controllers/dashboard_controller.rb]: Done.

File: app/controllers/dashboard_controller.rb (hunk 1 above)
Comment (adnanhemani) [reply app/controllers/dashboard_controller.rb]: Done.

File: app/controllers/dashboard_controller.rb (hunk 3 above)
Comment (adnanhemani) [reply app/controllers/dashboard_controller.rb]: You're right. Refactored.

File: app/models/project.rb (hunk 4 above)
Comment (adnanhemani) [reply app/models/project.rb]: Done

**Comment by an-ju** [comment]
LGTM.

**Pull request merged**